### PR TITLE
Add tests, PO, Core, Allure, Log and settings

### PR DESCRIPTION
**Pull request by barabule4ka**
Add tests, Core, Allure, Log and settings

### CODE REVIEW

File: MyFinalProject/Tests/LoginTest.cs [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+        [Category("Login")]
+        public void LoginAsUnknownUser()
+        {
+            var expectedError = "Authentication failed";
+            var user = UserBuilder.LoginAsFakeUser();
+
+            var page = new LoginPage();
+            page.OpenPage();
+            page.TryToLogin(user);
+           // string text = page.FindErrorMessage();
```
Comment (VladimirVolkovVD) [inline MyFinalProject/Tests/LoginTest.cs]: delete comment

File: MyFinalProject/Tests/LoginTest.cs [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        [Description("Use real user")]
+        [AllureOwner("Barabule4ka")]
+        [AllureSuite("Prestashop")]
+        [AllureSubSuite("Smoke cases")]
+        [AllureTms("TFS_MTS")]
+        [AllureIssue("issue-12345")]
+        [Category("Login")]
+        public void LoginAsRealUser()
+        {
+            var expectedUrl = "http://prestashop.qatestlab.com.ua/ru/my-account";
```
Comment (VladimirVolkovVD) [inline MyFinalProject/Tests/LoginTest.cs]: put this url to the page object for this page

**Review by VladimirVolkovVD — APPROVED** [review]

File: MyFinalProject/Tests/LoginTest.cs (hunk 1 above)
Comment (barabule4ka) [reply MyFinalProject/Tests/LoginTest.cs]: deleted

File: MyFinalProject/Tests/LoginTest.cs (hunk 2 above)
Comment (barabule4ka) [reply MyFinalProject/Tests/LoginTest.cs]: done

**Pull request merged**